whatsnext/models.py

```diff
@@ -1,34 +1,17 @@
 from __future__ import unicode_literals
 
 from django.db import models
+import datetime
 
-class Task(models.Model):
-    id = models.IntegerField(primary_key=True)
-    user_id = models.ForeignKey(User, on_delete=models.CASCADE)
-    task_description = models.CharField(max_length=1000)
-    priority = models.IntegerField()
-    # Later probably want to have a separate categories table to allow
-    # for users to create their own categories.
-    # But for now just have dumb integer field to keep the dream alive.
-    category = models.IntegerField()
-    created = models.DateField()
-    status = models.IntegerField()
+STATUSES = {
```

Comment:
Maybe this should be part of the Tasks model rather than a global? But no big deal either way.